src/js/utils/merge-options.js

```diff
@@ -3,46 +3,44 @@
  */
 import merge from 'lodash-compat/object/merge';
 
-function isPlain(obj) {
+const isPlain = function(obj) {
```

Comment:
Ok no problem, but if we're going by the Airbnb guid then we should move towards declarations.
https://github.com/airbnb/javascript#7.1
I've been using them for the smaller module functions at least.
https://github.com/videojs/video.js/blob/master/src/js/utils/format-time.js